pkg/cmd/pulumi/org_search.go

```diff
@@ -155,9 +168,18 @@ func renderSearchTable(w io.Writer, results []apitype.ResourceResult) error {
 		}
 		return err
 	}
-	return table.RenderTo(w)
+	err = table.RenderTo(w)
+	if err != nil {
+		return err
+	}
+	_, err = w.Write([]byte(urlInfo + "\n"))
```

Comment:
It would be good to have a test to go with this to confirm the formatting is as expected, it's not totally clear whether this should also have a "\n" before the url print.